CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# 0.7.0
+
+All the changes are documented on our [website](https://packit.dev).
+
+## Breaking changes
+
+* New option (`upstream_tag_template`) was added to the configuration file
```

Comment:
I don't understand how this is a breaking change, we've just made it configurable

Reply:
I chose the wrong line, sorry.